### PR TITLE
Added MintEvents

### PR DESCRIPTION
**Pull request by Codeindeed**

### CODE REVIEW

File: data/ecosystems/m/mintevents.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Ecosystem Level Information
+title = "MintEvents"
+
+sub_ecosystems = []
+
+github_organizations = ["https://github.com/Codeindeed"]
```
Comment (eherrerosj) [inline data/ecosystems/m/mintevents.toml:6]: not an org

**Review by eherrerosj — CHANGES_REQUESTED** [review]
left a comment, only github orgs and repos are accepted URLs

**Review by jubos — CHANGES_REQUESTED** [review]
Please see Enrique's comments above

**Comment by jubos** [comment]
Going to close this one out.  Please address the feedback above and reopen when it's ready.

**Pull request closed without merging**